app/models/publication-flow.js

```diff
@@ -46,57 +45,4 @@ export default class PublicationFlow extends Model {
     }
     return `${this.publicationNumber}`;
   }
-
-  get translationRequestsTotal() {
-    return this.publicationService.getPublicationCounts(this.id).then((totals) => {
-      const openVertalingen = this.publicationService.getPublicationCountsPerTypePerStatus(totals, CONFIG.ACTIVITY_TYPES.vertalen.url, CONFIG.ACTIVITY_STATUSSES.open.url);
-      const closedVertalingen = this.publicationService.getPublicationCountsPerTypePerStatus(totals, CONFIG.ACTIVITY_TYPES.vertalen.url, CONFIG.ACTIVITY_STATUSSES.closed.url);
-      const withdrawnVertalingen = this.publicationService.getPublicationCountsPerTypePerStatus(totals, CONFIG.ACTIVITY_TYPES.vertalen.url, CONFIG.ACTIVITY_STATUSSES.withdrawn.url);
-      const total = openVertalingen + closedVertalingen + withdrawnVertalingen;
-      return total;
-    });
-  }
-
-  get publishpreviewRequestsTotal() {
-    return this.publicationService.getPublicationCounts(this.id).then((totals) => {
```

Comment:
idem, mag mee opgekuist worden.
Deze gaat wel een micro-service aanspreken (lazy-loading).
Daar een beetje overwegen of we dat mee gaan opkuisen of laten staan voor later

Reply:
Ik had wat teveel opgekuist, dus ik heb deze terug gebracht. Mijn editor zei valselijk dat er geen usages waren.